lua/chadrc.lua

```diff
@@ -95,6 +95,13 @@ local M = {
             toggle_right = "<leader>v",
             toggle_bot = "<leader>h",
         },
+        unix_keymap = {
```

Comment:
Think it could be worth moving these to the `misc` table?

As `unix_keymap` isn't a plugin, and every other table above is a plugin, whereas `misc` has some non-plugin binds?

Or even better, a sub-table in `misc`?